code/frontend/src/styles/_themes.scss

```
@@ -7,9 +7,11 @@
 // Dark theme (default)
 :root,
 [data-theme='dark'] {
-  // Surfaces
+  // Surfaces with fallbacks for browsers that can't parse color-mix()
   --surface-ground:   #0c0614;
-  --surface-section:  #140b22;
+  --surface-ground:   color-mix(in srgb, var(--brand-500) 8%, #000000);
```

Stylelint (17.9.0)

[error] Duplicate custom property "--surface-ground" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)